clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/ResponseCodec.java

```diff
@@ -91,24 +89,13 @@
     .byteBuffer(MAP_VALUE_FIELD, 20)
     .build();
 
-  private final ChainCodec chainCodec;
-
-  public ResponseCodec() {
```

Comment:
I might be able to make this one static now too... but I didn't.  Should I have done?